Packs/Zscaler/.secrets-ignore

```diff
@@ -28,4 +28,6 @@ https://help.zscaler.com/zia/adding-cloud-nss-feeds-dns-logs
 https://help.zscaler.com/zia/adding-cloud-nss-feeds-admin-audit-logs
 https://help.zscaler.com/zia/nss-feed-output-format-dns-logs
 https://help.zscaler.com/zia/nss-feed-output-format-admin-audit-logs
-https://help.zscaler.com/zia/nss-feed-output-format-web-logs
\ No newline at end of file
+https://help.zscaler.com/zia/nss-feed-output-format-web-logs
+https://help.zscaler.com/zia/nss-feed-output-format-firewall-logs
+[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}[a-fA-F0-9\:]{1,5}
```

Comment:
Why is this in the secrets-ignore file? doesn't seem like it should be here, if its generating an error, maybe its a bug.